src/common/selectors/clusterSelectors.ts

```diff
@@ -57,14 +61,30 @@ export const allSubnetsIPv4 = (
   return !!networks?.every((network) => network.cidr && Address4.isValid(network.cidr));
 };
 
+export const hasIpv6Subnet = (
+  networks: (MachineNetwork | ClusterNetwork | ServiceNetwork)[] | undefined,
```

Comment:
The idea behind this module is that all the functions act on an instance of the cluster object :)

Reply:
That hasn't been the case for a while...
What would be best - rename the file to something like `helpers.ts` or `utils.ts` or move some of the functions to somewhere else?

Reply:
[SRP](https://en.wikipedia.org/wiki/Single-responsibility_principle), each module must have only one duty. `clusterSelectors.ts` should standardize the way we read data out of the Cluster model. Having files names `utils/helpers` all over the place makes it difficult to track what their scope is.
So if it hasn't been the case for a while, now is a good opportunity to improve it (as long as it doesn't require refactoring the whole code base, of course)